[2018-12-28 23:29:45,074] {models.py:1361} INFO - Dependencies all met for <TaskInstance: leetCoode_scrapying_dag.run_shell 2018-12-28T23:29:34.826690+00:00 [queued]>
[2018-12-28 23:29:45,079] {models.py:1361} INFO - Dependencies all met for <TaskInstance: leetCoode_scrapying_dag.run_shell 2018-12-28T23:29:34.826690+00:00 [queued]>
[2018-12-28 23:29:45,080] {models.py:1573} INFO - 
--------------------------------------------------------------------------------
Starting attempt 1 of 0
--------------------------------------------------------------------------------

[2018-12-28 23:29:45,087] {models.py:1595} INFO - Executing <Task(BashOperator): run_shell> on 2018-12-28T23:29:34.826690+00:00
[2018-12-28 23:29:45,088] {base_task_runner.py:118} INFO - Running: ['bash', '-c', 'airflow run leetCoode_scrapying_dag run_shell 2018-12-28T23:29:34.826690+00:00 --job_id 12 --raw -sd DAGS_FOLDER/scraping_scheduler_dag.py --cfg_path /var/folders/fz/pxkgsp6j3d946l3f71v3sj_00000gn/T/tmp32oprb_7']
[2018-12-28 23:29:46,160] {base_task_runner.py:101} INFO - Job 12: Subtask run_shell [2018-12-28 23:29:46,159] {__init__.py:51} INFO - Using executor SequentialExecutor
[2018-12-28 23:29:46,374] {base_task_runner.py:101} INFO - Job 12: Subtask run_shell [2018-12-28 23:29:46,374] {models.py:271} INFO - Filling up the DagBag from /Users/zhonghengli/Documents/Insight/leetCodeProject/leetcode_score_monitoring/src/airflow/airflow_home/dags/scraping_scheduler_dag.py
[2018-12-28 23:29:46,452] {base_task_runner.py:101} INFO - Job 12: Subtask run_shell [2018-12-28 23:29:46,452] {cli.py:484} INFO - Running <TaskInstance: leetCoode_scrapying_dag.run_shell 2018-12-28T23:29:34.826690+00:00 [running]> on host zhonghengsmbp2.fios-router.home
[2018-12-28 23:29:46,471] {bash_operator.py:74} INFO - Tmp dir root location: 
 /var/folders/fz/pxkgsp6j3d946l3f71v3sj_00000gn/T
[2018-12-28 23:29:46,472] {bash_operator.py:87} INFO - Temporary script location: /var/folders/fz/pxkgsp6j3d946l3f71v3sj_00000gn/T/airflowtmp77rfxobs/run_shell51378wlk
[2018-12-28 23:29:46,472] {bash_operator.py:97} INFO - Running command: sh $LEET_CODE_PROJECT_PATH/src/Scraper/run.sh 
[2018-12-28 23:29:46,484] {bash_operator.py:106} INFO - Output:
[2018-12-28 23:29:46,496] {bash_operator.py:110} INFO - sh: /src/Scraper/run.sh: No such file or directory
[2018-12-28 23:29:46,497] {bash_operator.py:114} INFO - Command exited with return code 127
[2018-12-28 23:29:46,505] {models.py:1760} ERROR - Bash command failed
Traceback (most recent call last):
  File "/Users/zhonghengli/Documents/Insight/leetCodeProject/Virtualenvs/scrapy_env/lib/python3.6/site-packages/airflow/models.py", line 1659, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/Users/zhonghengli/Documents/Insight/leetCodeProject/Virtualenvs/scrapy_env/lib/python3.6/site-packages/airflow/operators/bash_operator.py", line 118, in execute
    raise AirflowException("Bash command failed")
airflow.exceptions.AirflowException: Bash command failed
[2018-12-28 23:29:46,509] {models.py:1791} INFO - Marking task as FAILED.
[2018-12-28 23:29:46,524] {base_task_runner.py:101} INFO - Job 12: Subtask run_shell Traceback (most recent call last):
[2018-12-28 23:29:46,524] {base_task_runner.py:101} INFO - Job 12: Subtask run_shell   File "/Users/zhonghengli/Documents/Insight/leetCodeProject/Virtualenvs/scrapy_env/bin/airflow", line 32, in <module>
[2018-12-28 23:29:46,524] {base_task_runner.py:101} INFO - Job 12: Subtask run_shell     args.func(args)
[2018-12-28 23:29:46,524] {base_task_runner.py:101} INFO - Job 12: Subtask run_shell   File "/Users/zhonghengli/Documents/Insight/leetCodeProject/Virtualenvs/scrapy_env/lib/python3.6/site-packages/airflow/utils/cli.py", line 74, in wrapper
[2018-12-28 23:29:46,524] {base_task_runner.py:101} INFO - Job 12: Subtask run_shell     return f(*args, **kwargs)
[2018-12-28 23:29:46,525] {base_task_runner.py:101} INFO - Job 12: Subtask run_shell   File "/Users/zhonghengli/Documents/Insight/leetCodeProject/Virtualenvs/scrapy_env/lib/python3.6/site-packages/airflow/bin/cli.py", line 490, in run
[2018-12-28 23:29:46,525] {base_task_runner.py:101} INFO - Job 12: Subtask run_shell     _run(args, dag, ti)
[2018-12-28 23:29:46,525] {base_task_runner.py:101} INFO - Job 12: Subtask run_shell   File "/Users/zhonghengli/Documents/Insight/leetCodeProject/Virtualenvs/scrapy_env/lib/python3.6/site-packages/airflow/bin/cli.py", line 406, in _run
[2018-12-28 23:29:46,525] {base_task_runner.py:101} INFO - Job 12: Subtask run_shell     pool=args.pool,
[2018-12-28 23:29:46,525] {base_task_runner.py:101} INFO - Job 12: Subtask run_shell   File "/Users/zhonghengli/Documents/Insight/leetCodeProject/Virtualenvs/scrapy_env/lib/python3.6/site-packages/airflow/utils/db.py", line 74, in wrapper
[2018-12-28 23:29:46,525] {base_task_runner.py:101} INFO - Job 12: Subtask run_shell     return func(*args, **kwargs)
[2018-12-28 23:29:46,525] {base_task_runner.py:101} INFO - Job 12: Subtask run_shell   File "/Users/zhonghengli/Documents/Insight/leetCodeProject/Virtualenvs/scrapy_env/lib/python3.6/site-packages/airflow/models.py", line 1659, in _run_raw_task
[2018-12-28 23:29:46,525] {base_task_runner.py:101} INFO - Job 12: Subtask run_shell     result = task_copy.execute(context=context)
[2018-12-28 23:29:46,525] {base_task_runner.py:101} INFO - Job 12: Subtask run_shell   File "/Users/zhonghengli/Documents/Insight/leetCodeProject/Virtualenvs/scrapy_env/lib/python3.6/site-packages/airflow/operators/bash_operator.py", line 118, in execute
[2018-12-28 23:29:46,525] {base_task_runner.py:101} INFO - Job 12: Subtask run_shell     raise AirflowException("Bash command failed")
[2018-12-28 23:29:46,525] {base_task_runner.py:101} INFO - Job 12: Subtask run_shell airflow.exceptions.AirflowException: Bash command failed
[2018-12-28 23:29:50,065] {logging_mixin.py:95} INFO - [2018-12-28 23:29:50,064] {jobs.py:2627} INFO - Task exited with return code 1
